README.rst

```diff
@@ -570,12 +570,12 @@ page or directly downloaded using the following URL::
 Sensors
 =======
 
-Agents (a.k.a. "Sensors") check in to the Dalton server frequently
+Agents (a.k.a. "Sensors") check into the Dalton server frequently
```

Comment:
correcting `in to` to `into` has been fairly controversial, but I believe this is correct.